cfspopcon/formulas/separatrix_operational_space/alpha_t.py

```diff
@@ -0,0 +1,87 @@
+"""Routines to calculate the alpha_t turbulence parameter."""
+import numpy as np
+
+from ...unit_handling import Quantity, Unitfull, convert_units, ureg, wraps_ufunc
+
+
+def calc_alpha_t(
+    electron_density: Unitfull,
+    electron_temperature: Unitfull,
+    edge_safety_factor: Unitfull,
+    major_radius: Unitfull,
+    ion_mass: Unitfull,
+    Zeff: Unitfull,
+    Z: Unitfull,
+    ion_to_electron_temperature_ratio: float = 1.0,
+) -> Unitfull:
+    """Calculate the turbulence parameter alpha_t.
+
+    Equivalent to alpha_t = C * omega_B
+    where C = normalized collisionality and omega_B = curvature drive term
```

Comment:
https://github.com/cfs-energy/cfspopcon/pull/12#discussion_r1465404544

Missing reference.

How does this relate to calc_normalised_collisionality? Could that be used here instead?